persistence/src/main/java/com/epam/esm/persistence/dao/certificate/CertificateDAO.java

```diff
@@ -1,24 +1,15 @@
 package com.epam.esm.persistence.dao.certificate;
```

Comment:
I guess you can remove certificate/tag etc. packages in dao, as right now each of them contain only one class

Reply:
done